hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/StoreWorker.java

```diff
@@ -71,6 +69,7 @@
      * @see #calculateHighestStoreTime
      */
     private long lastHighestStoreTime;
+    private boolean stopped;
```

Comment:
I prefer to call these variables `running`, because we mostly use them like `while (runnning) { ... }`. So we don't have to negate them all them time.